src/nvim/lua/executor.c

```diff
@@ -546,6 +611,13 @@ Object executor_exec_lua_cb(LuaRef ref, const char *name, Array args,
   }
 }
 
+/// check if the current execution context is safe for calling deferred API
+/// methods. Luv callbacks are unsafe as they are called inside the uv loop.
+bool nlua_check_safe_cb(lua_State *lstate)
```

Comment:
```suggestion
bool nlua_is_deferred_safe(lua_State *lstate)
```

Usually `_cb` suffix means the thing is itself a callback, no?